examples/case_studies/lazy_load/index.xml

```diff
@@ -4,7 +4,7 @@
       <style id="Body"
              backgroundColor="white"
              flex="1"
-             paddingTop="48" />
+              />
```

Comment:
I think the `<body>` tag lower down this file should also set `safe-area` attribute

Reply:
The design of this screen is different from the others, it feature big colorful blocks that should not use `safe-area`.